packages/core/package.json

```diff
@@ -4218,23 +4218,31 @@
         "clean": "ts-node ../../scripts/clean.ts dist/",
         "copyFiles": "ts-node ./scripts/build/copyFiles.ts",
         "buildScripts": "npm run generateClients && npm run generatePackage && npm run copyFiles",
-        "compile": "npm run clean && npm run buildScripts && tsc -p ./ && webpack --mode development",
-        "compileLite": "tsc -p ./",
-        "watch": "npm run clean && npm run buildScripts && tsc -watch -p ./",
-        "testCompile": "npm run clean && npm run buildScripts && tsc -p ./",
+        "compile": "npm run testCompile && webpack",
+        "compileOnly": "tsc -p ./",
+        "webpackDevelopment": "webpack --mode development",
+        "serveVue": "webpack serve --config-name vue --mode development",
+        "watch": "npm run testCompile -- -- --watch",
+        "testsWebWatch": "npm run clean && npm run buildScripts && npm run webpackDevelopment -- --watch",
+        "testCompile": "npm run clean && npm run buildScripts && npm run compileOnly",
         "test": "npm run testCompile && c8 ts-node ./scripts/test/test.ts",
-        "webCompile": "npm run clean && npm run buildScripts && webpack --config-name web",
-        "webWatch": "npm run webCompile -- --watch",
-        "testWeb": "npm run webCompile && c8 ts-node ./scripts/test/testWeb.ts",
+        "webDevelopmentCompile": "npm run compile -- --mode development",
+        "testWeb": "npm run webDevelopmentCompile && c8 ts-node ./scripts/test/testWeb.ts",
         "testE2E": "npm run testCompile && c8 ts-node ./scripts/test/testE2E.ts",
         "testInteg": "npm run testCompile && c8 ts-node ./scripts/test/testInteg.ts",
         "format": "prettier --ignore-path ../../.prettierignore --check src scripts",
         "formatfix": "prettier --ignore-path ../../.prettierignore --write src scripts",
         "lint": "ts-node ./scripts/lint/testLint.ts && npm run format",
         "generateClients": "ts-node ./scripts/build/generateServiceClient.ts ",
         "generatePackage": "ts-node ./scripts/build/generateIcons.ts",
-        "generateTelemetry": "node ../../node_modules/@aws-toolkits/telemetry/lib/generateTelemetry.js --extraInput=src/shared/telemetry/vscodeTelemetry.json --output=src/shared/telemetry/telemetry.gen.ts",
-        "serve": "webpack serve --config-name vue-hmr --mode development"
+        "generateTelemetry": "node ../../node_modules/@aws-toolkits/telemetry/lib/generateTelemetry.js --extraInput=src/shared/telemetry/vscodeTelemetry.json --output=src/shared/telemetry/telemetry.gen.ts"
+    },
+    "scriptsComments": {
```

Comment:
is this an NPM convention (i.e. does it feed into tools like vscode) or just something we are doing?

Reply:
I saw this convention on stackoverflow. My understanding is that VSCode knows which package.json keys it cares about and ignores the rest. So adding `*Comments` shouldn't match any existing keys and wont be an issue

Reply:
> VSCode knows which package.json keys it cares about and ignores the rest.

That's actually part of the package.json spec. I was just curious if this particular key was actually used by any tooling.